views/pages/guides/runtime-environments.hbs

```diff
@@ -99,7 +99,7 @@ if (global.Intl) {
     if (!areIntlLocalesSupported(localesMyAppSupports)) {
         // `Intl` exists, but it doesn't have the data we need, so load the
         // polyfill and replace the constructors with need with the polyfill's.
-        require('intl');
+        var IntlPolyfill = require('intl');
```

Comment:
I didn't see any error. Just reading the code, it was unclear as where the `IntlPolyfill` was coming from.

What is the use of a global variable here ? Global vars are considered... (you know what I mean)